git/foundations_git/commit_messages.md

```diff
@@ -20,14 +20,56 @@ Yes! Let me give you a quick list of reasons why:
 
 -   Having a good commit message history will also be helpful if you come back to a project you were working on after stepping away from it for a while. You likely won't remember your thought process and changes made when initially writing out the code. 
 
+### Bad vs. Good Commits
+
+When it comes to writing commits, it is crucial to know how to write them effectively. Here's an example of a bad commit message:
+
+~~~
+fix a bug
+~~~
+
+Even though it describes what you did, the message is too vague, which leaves the other developers on your team confused. A good commit message will explain the **why** behind your changes. In other words, a commit message describes what problem your changes solve and how it solves them. 
+
+Effective commits consist of two separate parts: a subject, and a body:
+
+#### Subject
+A brief summary of the change you made to the project.  **Note:** GitHub has a 72-character limit so we recommend keeping your commits' subject to within this amount. 
+
+`This is the change I made to the codebase.`
+
+#### Body
+A concise yet clear description of what you did.  
+
+~~~
+Describe the problem your commit solves and how.
+~~~
+
+
+Now that we learned the secret to creating a good commit message, let's try and fix the commit message from earlier:
+
+~~~git
+Add missing link and alt text to the company's logo
+
+Screen readers won't read the images to users with disabilities without this information
+~~~
+`Screen readers won't read the images to users with disabilities without this information`
+
+Ahh, that's better! :) Now, developers can gain a better understanding of this commit message because it does the following:
```

Comment:
Forgot to comment on this the first run through, but this breakdown is a really nice touch 🏆

Reply:
You're welcome! :) I want to stay true to your casual and humorous writing style(that's my approach to writing about tech by the way 😆)